yt_dlp/extractor/moviepilot.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import unescapeHTML
+
+
+class MoviepilotIE(InfoExtractor):
+    """Downloader for the trailer for a movie."""
+    _VALID_URL = r'https?://(?:www\.)?moviepilot\.de/movies/(?P<id>[^/]+)/?.*'
+
+    _TESTS = [{
+        'url': 'https://www.moviepilot.de/movies/interstellar-2/',
+        'info_dict': {
+            'id': 'interstellar-2',
+            'title': 'Interstellar'
+        },
+        'playlist_count': 1,
+    }, {
+        'url': 'https://www.moviepilot.de/movies/interstellar-2',
+        'only_matching': True,
+    }, {
+        'url': 'https://www.moviepilot.de/movies/interstellar-2/kinoprogramm/berlin',
+        'info_dict': {
+            'id': 'interstellar-2',
+            'title': 'Interstellar'
+        },
+        'playlist_count': 1,
+    }, {
+        'url': 'https://www.moviepilot.de/movies/queen-slim/trailer',
+        'info_dict': {
+            'id': 'queen-slim',
+            'title': 'Queen & Slim'  # test escaping
+        },
+        'playlist_count': 1,
+    }, {
+        'url': 'https://www.moviepilot.de/movies/der-geiger-von-florenz/trailer',
+        'info_dict': {
+            'id': 'der-geiger-von-florenz',
+            'title': ' Der Geiger von Florenz'
+        },
+        'skip': 'No trailer for this movie',
+        'playlist_count': 0,
+    }, {
+        'url': 'https://www.moviepilot.de/movies/muellers-buero/',
+        'info_dict': {
+            'id': 'muellers-buero',
+            'title': 'Müllers Büro'  # test umlauts
+        },
+        'playlist_count': 1,
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        # normalize url
+        url = f"https://www.moviepilot.de/movies/{video_id}/trailer"
+
+        webpage = self._download_webpage(url, video_id)
+
+        title = unescapeHTML(self._search_regex(
+            r'<meta property="og:title" content="(.*)"', webpage, 'title'))
+        embed_url = self._search_regex(r'meta itemprop="embedURL" content="(.*?)"', webpage, 'embed_url')
+
+        return self.playlist_result([self.url_result(embed_url)],
+                                    playlist_id=video_id,
+                                    playlist_title=title)
```

Comment:
why playlist? You should just return the URL

Reply:
This was my workaround to trigger the next extractor. This is also the reason why I have no further metadata: Because they are meaningless in playlists.

I have found about `_type: 'url_transparent'` which seems much more suited and updated the extractor with some metadata.